portfolio/src/main/webapp/script.js

```diff
@@ -45,13 +45,15 @@ async function getCommentFromFetch() {
     const response = await fetch("/data");
     const comments = await response.json();
     const commentContainer = document.getElementById("comment-container");
+    const maxComments = parseInt(document.getElementById("comment-limit").value, 10);
```

Comment:
It's hard, but not impossible, to insert non-numeric data into a `number`-type `input` element. In those cases, `parseInt` returns something besides a number, and that case should be handled here.

Reply:
I will handle the case by using the Number.isNaN() function.